pandas/core/indexing.py

```diff
@@ -1,8 +1,9 @@
 import textwrap
-from typing import Tuple
+from typing import TYPE_CHECKING, Any, List, Optional, Tuple, Union, cast
 import warnings
 
 import numpy as np
+from numpy import ndarray
```

Comment:
I think outside of cython we generally avoid this

Reply:
sure. will change.